action.yml

```
@@ -58,7 +58,7 @@ runs:
           EXCLUDE_GLOB="-exclude-glob=${{ inputs.exclude-glob }}"
```

Can you please add the quotes to the `exclude-glob` as well?

```suggestion
          EXCLUDE_GLOB="-exclude-glob='${{ inputs.exclude-glob }}'"
```